### PR TITLE
Create a stub of anuragkum's recommendations

### PR DESCRIPTION
**Pull request by anuragkum**
This is my extended description of my methods!

### CODE REVIEW

**Comment by yaning98** [comment]
This is Yaning

File: anuragkum.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+# San Diego, CA
+
+### Places to eat
+
+- Aviator's Smokehouse
+- Cooley's
```
Comment (yaning98) [inline anuragkum.md:7]: This is wrong line

File: anuragkum.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+# San Diego, CA
```
Comment (yaning98) [inline anuragkum.md:2]: delete this line

File: anuragkum.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (anuragkum) [inline anuragkum.md:3]: delete line 3 white space

**Pull request merged**